src/components/common/inside/cbrum/cbrumIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import 'components/common/inside/cbrum/cbrumIcon.css'
+
+import userIcon from 'assets/images/usericon.jpeg'
+
+const CbrumIcon = () => {
+  return(
+    <div className='cbrum-row'>
+      <div>
+        <img src={userIcon} alt="userIcon" className='panel-header--user-img' />
+        <h4 className=''>cbrum</h4>
```

Comment:
If you are not adding any classNames, it's recommended to remove this instead of leaving empty strings.
```suggestion
        <h4>cbrum</h4>
```